docs/superpowers/plans/2026-05-08-item-detail-image-stretch-redesign.md

```
@@ -0,0 +1,399 @@
+    _scrollController.dispose();
+    _returnAnim.removeListener(_onReturnAnimTick);
+    _returnAnim.dispose();
+    _stretchScaleVN.dispose();
+    super.dispose();
+  }
+```
+
+- [ ] **Step 5: 포매팅 + 컴파일 검증**
+
+내부망 환경 — `dart pub get` 불가. `dart format`만 실행:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)